common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -110,7 +110,6 @@ function (HTML5Video, Resizer) {
         state.videoPlayer.currentTime = 0;
 
         state.videoPlayer.goToStartTime = true;
-        state.videoPlayer.stopAtEndTime = true;
```

Comment:
Do we actually want to remove this line (and related) or should we just set it to false?